resources/test/fixtures/B026.py

```diff
@@ -0,0 +1,21 @@
+"""
```

Comment:
<img width="614" alt="image" src="https://user-images.githubusercontent.com/17039389/201139747-0132848b-641c-4cbc-8d51-3c48222d143e.png">

The result looks correct.